README.md

```diff
@@ -125,8 +125,23 @@ class E2eTest extends PantherTestCase
         $client->request('GET', '/mypage');
 
         // Use any PHPUnit assertion, including the ones provided by Symfony
-        $this->assertPageTitleContains('My Title');
-        $this->assertSelectorTextContains('#main', 'My body');
+        self::assertPageTitleContains('My Title');
```

Comment:
We use `$this->` in Symfony: https://symfony.com/doc/current/testing/functional_tests_assertions.html

Reply:
Ok to change but can you explain me why this choice since those function are `static` ?

Reply:
It's historical IIRC.